x/valset-pref/types/keys.go

```diff
@@ -7,6 +7,9 @@ var (
 	// StoreKey defines the primary module store key
 	StoreKey = ModuleName
 
+	// RouterKey is the message route for slashing
+	RouterKey = ModuleName
```

Comment:
Is this really for slashing or was it from copy paste? Genuinely asking since its possible

Reply:
This was a copy-pasta from other module